docs/antora.yml

```diff
@@ -10,5 +10,6 @@ asciidoc:
     db-vector: 'Astra DB Serverless'
     db-serverless: 'Astra DB Serverless'
     astra-ui: 'Astra Portal'
+    graph-store: 'Knowledge Store'
```

Comment:
I think we're moving towards calling this "Graph Store". Not sure whether we want to reflect that here / already.